collect_app/src/main/java/org/odk/collect/android/gdrive/InstanceGoogleSheetsUploader.java

```diff
@@ -116,6 +116,7 @@ public String uploadOneSubmission(Instance instance, String spreadsheetUrl) thro
 
             TreeElement instanceElement = getInstanceElement(formFilePath, instanceFile);
             setUpSpreadsheet(spreadsheetUrl);
+            sheetsHelper.updateSpreadsheetLocale(spreadsheet.getSpreadsheetId());
```

Comment:
Does this end up changing the locale for a sheet with existing submissions?

Reply:
That method checks if it's a new spreadsheet and only then updates the locale.